src/protocols/ethereum/views/TransferSendModal.vue

```diff
@@ -0,0 +1,133 @@
+<!-- eslint-disable vue/no-v-model-argument -->
+<template>
+  <TransferSendBase
+    v-bind="$attrs"
+    :protocol="PROTOCOL_ETHEREUM"
+    :current-step="currentStep"
+    :edit-transfer="editTransfer"
+    :proceed-to-next-step="proceedToNextStep"
+    :primary-button-disabled="error || !transferData.address || !transferData.amount"
+    @close="resolve"
+  >
+    <template #content>
+      <component
+        :is="currentStepConfig.component"
+        ref="currentRenderedComponent"
+        v-model:transferData="transferData"
+        @success="currentStepConfig.onSuccess"
+        @error="(val) => error = val"
+      />
+    </template>
+  </TransferSendBase>
+</template>
+
+<script lang="ts">
+import {
+  Component,
+  computed,
+  defineComponent,
+  PropType,
+  ref,
+} from 'vue';
+import { Encoded } from '@aeternity/aepp-sdk';
+import type {
+  ResolveCallback,
+  TransferFormModel,
+  TransferSendStep,
+} from '@/types';
+import {
+  useBalances,
+  useCurrencies,
+} from '@/composables';
+
+import TransferSendBase from '@/popup/components/Modals/TransferSendBase.vue';
+import {
+  PROTOCOL_ETHEREUM,
+  PROTOCOL_VIEW_TRANSFER_SEND,
+  TRANSFER_SEND_STEPS,
+} from '@/constants';
```

Comment:
I suggest to move the constants right after the types import. In general I think the order of the imports should go from less specific to more specific:
```
1. External deps
2. Types
3. Constants
4. Utils
5. Libs
6. Composables
7. Protocol related stuff
8. Components
9. Protocol related components
```